LawnCare/android/app/src/main/java/com/redflag/lawncare/MainActivity.java

```diff
@@ -42,6 +42,7 @@ void setTabsAndViewPager() {
             tabLayout.addTab(tabLayout.newTab().setText("Book"));
             tabLayout.addTab(tabLayout.newTab().setText("FAQ"));
             tabLayout.addTab(tabLayout.newTab().setText("Contact"));
+            tabLayout.addTab(tabLayout.newTab().setText("Quote"));
```

Comment:
These labels should be translated. Something we missed in the last version. I will add a ticket in the future to fix these

Reply:
I couldnt find where the tabs id are, will add them later when I find it! Let me know if you know~